ui/shared/components/buttons/SearchResultsLink.jsx

```diff
@@ -48,15 +48,21 @@ const ConnectedSearchResultsLink = connect(null, mapDispatchToProps)(SearchResul
 
 export default ConnectedSearchResultsLink
 
-const getGeneSearchProps = af => ({
-  inheritance: { mode: ANY_AFFECTED },
-  freqs: FREQUENCIES.filter(({ name }) => name !== THIS_CALLSET_FREQUENCY && name !== SV_CALLSET_FREQUENCY).reduce(
-    (acc, { name }) => ({ ...acc, [name]: { af } }), {},
-  ),
+const getGeneSearchProps = (af, af2) => ({
+  inheritance: { mode: ANY_AFFECTED, filter: {} },
+  freqs: {
+    ...FREQUENCIES.filter(({ name }) => name !== THIS_CALLSET_FREQUENCY && name !== SV_CALLSET_FREQUENCY).reduce(
```

Comment:
I didn't catch it when the ticket was made, but the intention here was not to have different frequency handling for different popualtions. Lynn has now clarified on the ticket that we should not filter on topmed at all, and everything else in `FREQUENCIES` is either a gnomad or a callset frequency and should have the new 0.03 value. It would be cleaner therefore to filter for "not topmed" 

Also, I'm still waiting for the cnfirmation form Lynn, but I think we can get rid of the ` RareGeneSearchLink` entirely (which is only used in `PhenotypePrioritizedGenes`) and have consistant gene search behavior across seqr